__tests__/app.test.js

```diff
@@ -90,6 +90,22 @@ describe('GET /api/reviews', () => {
     });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+    test('returns an object of the posted comment', () => {
+        return request(app).post('/api/reviews/5/comments').send({ username: 'philippaclaire9', body: 'I hate this game, waste of my time'}).expect(201).then(response => {
+            const comment = response.body.comment;
+            expect(comment).toMatchObject([{
+                author: 'philippaclaire9', 
+                body: 'I hate this game, waste of my time',
+                comment_id: 7,
+                created_at: expect.any(String), 
+                review_id: 5, 
+                votes: 0
+            }])
```

Comment:
This comment shouldn't be returned in an array at the end - it's always going to be just one object